modules/apps/commerce/commerce-product-definitions-web/src/main/java/com/liferay/commerce/product/definitions/web/internal/display/context/CPDefinitionsDisplayContext.java

```diff
@@ -142,6 +143,26 @@ public CreationMenu getAccountGroupsCreationMenu() throws PortalException {
 		).build();
 	}
 
+	public List<DropdownItem> getBulkActionDropdownItems() {
+		List<DropdownItem> bulkActions = new ArrayList<>();
+
+		RenderResponse renderResponse = cpRequestHelper.getRenderResponse();
+
+		ActionURL actionURL = renderResponse.createActionURL();
+
+		actionURL.setParameter(
+			ActionRequest.ACTION_NAME, "editProductDefinition");
```

Comment:
Here also, I think it should be this, not 100% sure though
```suggestion
			ActionRequest.ACTION_NAME, "/cp_definitions/edit_cp_definition");
```